### ISSUE TITLE
Balance RPCs return empty balance entries

### ISSUE THREAD

**Issue by dexX7** [OmniLayer/omnicore#257]
When using `"omni_getallbalancesforid"` or `"omni_getallbalancesforaddress"`, then there may be entries with zero balances. Those are entries, which previously had some balance, but now have none.  

I think the entries should either be filtered out, or removed (but not on the RPC layer).

Empty balance entries are not persisted, so this only affects the runtime state.

Related: https://github.com/OmniLayer/OmniJ/pull/116#issuecomment-146936033

**Comment by zathras-crypto** [comment]
> Those are entries, which previously had some balance, but now have none.

**Comment by dexX7** [comment]
> - if we want to filter tally objects it's the RPC layer we should be doing it.
> - if we want to remove tally objects it's the tally itself we should be doing it.

This is what I was referring to.

Let me rephrase: it probably would be not a good, if we start to remove/delete tally entries directly in the RPCs. Filtering can be done on the RPC layer though. :)

I'd say: let's filter them for now.

**Comment by zathras-crypto** [comment]
Would this suffice? https://github.com/OmniLayer/omnicore/pull/262

**Comment by dexX7** [comment]
Resolved via #262. :)
